Add Jan 2020 newsletter (#787)
alan-turing-institute/the-turing-way: communications/newsletters/newsletter_09_Jan2020.md

This year, among [18 SSI fellowship](https://software.ac.uk/about/fellows?_ga=2.125569866.1329264381.1579011146-1585188985.1568709796) recipients are several our valuable contributors. They include two of our core members, Sarah Gibson and Jez Cope. 
Sarah aims to [build a welcoming community of reproducible research](https://software.ac.uk/about/fellows/sarah-gibson) by encouraging contributions, promoting diversity and transparency, and fostering collaborations. 
Jez is interested in developing [infrastructure and practices to archive and preserve research data and software for the future](https://software.ac.uk/about/fellows/jez-cope). SSI fellowships are awarded every year to open science advocates for supporting their effort as ambassadors in their respective fields. More details in [this blog post](https://software.ac.uk/blog/2020-01-10-announcing-2020-software-sustainability-institute-fellows).
Two more fellows, [Alex Clarke](https://software.ac.uk/about/fellows/alex-clarke) and [Will Hulme](https://software.ac.uk/about/fellows/will-hulme), are contributors of _The Turing Way_ project at the Book Dash events in 2019.

> We also extend our wishes to [Tarek Allam](https://software.ac.uk/about/fellows/tarek-allam) who is a members of The Turing Institute. 

Reviewer: Typo :)

```suggestion
We also extend our wishes to [Tarek Allam](https://software.ac.uk/about/fellows/tarek-allam) who is a member of The Turing Institute. 
```

Author: Thanks for the edit Sarah. 

The deadline will depend on @KirstieJane's availability as she schedules and send the newsletter out. Since the collab cafe is tomorrow, it would be nice to send out by tomorrow morning.

It is an in-person collaborative event on “how to data science” resource development, where you will work with others to add to and improve [the Turing Way book](https://github.com/alan-turing-institute/the-turing-way). 

This event will start on the evening of 20 February 2020 with an introduction to the project, which will be followed by an official dinner. 
The complete day of 21 February 2020 will be dedicated to the project content development. 
Your contributions may include, but are not limited to: science communication - skills in writing, editing and/or exciting interactive communications, great examples and case studies of reproducible research, writing new chapters, editing the current chapters and updating how the book looks online (CSS, logos, layout etc). 

## News from the community

***Boost your research reproducibility with Binder workshops in Cambridge***
 

> [The British Antarctic Survey](www.bas.ac.uk) is hosting a half-day workshop on 31 January 2020 from 13:30 to 17:00 on research reproducibility in The Turing Way project. 

Reviewer 2: ```suggestion
[The British Antarctic Survey](https://www.bas.ac.uk/) is hosting a half-day workshop on 31 January 2020 from 13:30 to 17:00 on research reproducibility in The Turing Way project. 
```

Reviewer 2: The link originally didn't work from me - GitHub interprets it as a relative file without the prefix.

At this workshop, the trainers will discuss reproducible computing environments, with examples of others’ projects in myBinder.org to help you learn how to prepare a binder-ready project. 
At the end of the workshop, you will be able to take some of your own content (in R, Jupyter notebook, or scripts that can be run in the terminal) and prepare it so that it can be used by others on myBinder.org. 
For registration, please contact Dr. Anita Faul from the Artificial Intelligence Lab of British Antarctic Survey by emailing [anfaul@bas.ac.uk](mailto:anfaul@bas.ac.uk).

> One more Binder workshop will be hosted by the Raspberry Pi Foundation in Cambridge on 29 January 2020. Sarah Gibson will be teaching at this workshop, details of which can be found [here](SARAH WILL ADD LINK HERE).

Reviewer: Adding PyData Cam Meetup link!

```suggestion
One more Binder workshop will be hosted by the Raspberry Pi Foundation in Cambridge on 29 January 2020. Sarah Gibson will be teaching at this workshop, details of which can be found [here](https://www.meetup.com/PyData-Cambridge-Meetup/events/267902974/).
```

Do you have your _The Turing Way_ stories and experiences to share? Contact us to learn how you can add them on our GitHub repository.

## Acknowledgements and celebrations

***Congratulations Software Sustainability Institute’s (SSI) Fellows 2020***

[Tweet](https://twitter.com/SoftwareSaved/status/1215572740381372416?s=20) from the official Twitter page of SSI.

This year, among [18 SSI fellowship](https://software.ac.uk/about/fellows?_ga=2.125569866.1329264381.1579011146-1585188985.1568709796) recipients are several of our valuable contributors. This fantastic set of fellows include two of our core members, Sarah Gibson and Jez Cope. 
Sarah aims to [build a welcoming community of reproducible research](https://software.ac.uk/about/fellows/sarah-gibson) by encouraging contributions, promoting diversity and transparency, and fostering collaborations. 
Jez is interested in developing [infrastructure and practices to archive and preserve research data and software for the future](https://software.ac.uk/about/fellows/jez-cope). 

> Two more fellows, [Alex Clarke](https://software.ac.uk/about/fellows/alex-clarke) and [Will Hulme](https://software.ac.uk/about/fellows/will-hulme), are contributors of _The Turing Way_ project at the Book Dash events in 2019.

Reviewer 2: ```suggestion
Three more fellows, [Alex Clarke](https://software.ac.uk/about/fellows/alex-clarke), [Will Hulme](https://software.ac.uk/about/fellows/will-hulme) and [Tarek Allam](https://software.ac.uk/about/fellows/tarek-allam), are contributors of _The Turing Way_ project at the Book Dash events in 2019.
```

> We also extend our wishes to [Tarek Allam](https://software.ac.uk/about/fellows/tarek-allam) who is a member of The Alan Turing Institute.

Reviewer 2: ```suggestion
```

Reviewer 2: I'm moving Tarek up to the book dash contributors, not deleting him! It wasn't entirely clear from Kirstie's tweet, but he was at the book dash in Manchester.

[Tweet](https://twitter.com/SoftwareSaved/status/1215572740381372416?s=20) from the official Twitter page of SSI.

This year, among 18 [SSI fellowship]() recipients are our two highly contributing members, Sarah Gibson and Jez Cope. We are really thrilled to have their work and effort recognized by this prestigious fellowship. 
Sarah aims to [build a welcoming community of reproducible research](https://software.ac.uk/about/fellows/sarah-gibson) by encouraging contributions, promoting diversity and transparency, and fostering collaborations. 
Jez is interested in developing [infrastructure and practices to archive and preserve research data and software for the future](https://software.ac.uk/about/fellows/jez-cope). SSI fellowships are awarded every year to open science advocates for supporting their effort as ambassadors in their respective fields. More details in [this blog post](https://software.ac.uk/blog/2020-01-10-announcing-2020-software-sustainability-institute-fellows).

> 

Reviewer 3: Should we also shout out to Tarek, Alex and William who attended book dashes like Kirstie did in her tweet? https://twitter.com/kirstie_j/status/1215672443785924610
I don't want to say that you need to attend a book dash to become a fellow, but it doesn't harm to highlight that book dash attendance is definitely beneficial 😄

Author: Ah, that's true. We definitely should! I like your argument <3

Author: Added!
